frontend/css/index.css

```diff
@@ -211,16 +226,6 @@ body {
     font-size: 10px;
 }
 
-.school-badge.magnet {
-    background-color: #FFAE8B;
-    border:2px solid rgba(255,255,255,0.5);
-}
-
-.school-badge.year-round {
-    background-color: #EF83A2;
-    border:2px solid rgba(255,255,255,0.5);
```

Comment:
The corresponding colors are still used for the [map icons](https://github.com/codefordurham/school-navigator/blob/master/frontend/css/index.css#L200-L206) borders I believe. Maybe we should get rid of those if we're doing away with the color coding?